data/org.eclipse.birt.core.script.function/src/org/eclipse/birt/core/script/function/bre/BirtComp.java

```diff
@@ -36,11 +36,6 @@
 {
 
 
-	/**
-	 * 
-	 */
-	private static final long serialVersionUID = 29055295217417372L;
```

Comment:
why do you remove this serial version UID? I think it was intendedly added.

Reply:
Because the class BirtComp does not implement Serializable (although maybe it did sometime in the past).

Reply:
These functions like BirtComp need to be serializable in some cases.
I would prefer to add the Serializable interface rather than remove the UID.

Reply:
I just added a commit that makes IScriptFunctionExecutors Serializable and adds the serialVersionUID accordingly